### DISCUSSION TITLE
Two forms getting submitted

### DISCUSSION THREAD

**Discussion by pchvramesh** [react-hook-form/react-hook-form#11435] [General]
Hi 
I have a situation where I'm showing a form on model pop up on top of  another form component.
When i click submit on model popup the behind form is also getting submitted.

How can i stop this.

**Comment by Moshyfawn** [comment]
[`event.stopPropagation();`](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation) should do it.
